publishes/pages/app/resources/{{ model }}Resource.php

```diff
@@ -32,16 +32,18 @@ public function toArray($request)
             'name' => $this->name,
             'slug' => $this->slug,
 
+            'template' => $this->template,
+
             // dynamic
-            'content' => $this->content->toArray(),
+            'content'    => $this->content->toArray(),
             'attributes' => $this->attributes,
```

Comment:
```suggestion
            'attributes' => $this->attributes->toArray(),
```
doesn't this require the `toArray()` anymore?